APMrover2/sailboat.h

```diff
@@ -96,6 +99,8 @@ class Sailboat
     AP_Float sail_heel_angle_max;
     AP_Float sail_no_go;
     AP_Float sail_windspeed_min;
+    AP_Float max_cross_track;
```

Comment:
let's put the max at the end of the variable name as well so let's call it cross_track_max or xtrack_max or something similar.

Reply:
fixed